src/Service/RemoteInstanceManager.php

```diff
@@ -57,11 +57,18 @@ public function updateInstance(Instance $instance, bool $force = false): bool
 
             $nodeInfoRaw = $this->client->fetchInstanceNodeInfo($linkToUse->href, false);
             $this->logger->debug('got raw nodeinfo for url {url}: {raw}', ['raw' => $nodeInfoRaw, 'url' => $linkToUse]);
-            /** @var NodeInfo $nodeInfo */
-            $nodeInfo = $serializer->deserialize($nodeInfoRaw, NodeInfo::class, 'json');
-
-            $instance->software = $nodeInfo?->software?->name;
-            $instance->version = $nodeInfo?->software?->version;
+            try {
+                /** @var NodeInfo $nodeInfo */
+                $nodeInfo = $serializer->deserialize($nodeInfoRaw, NodeInfo::class, 'json');
+                $instance->software = $nodeInfo?->software?->name;
+                $instance->version = $nodeInfo?->software?->version;
+            } catch (\Error|\Exception $e) {
+                $this->logger->warning('There as an exception decoding the nodeinfo from {url}: {e} - {m}', [
+                    'url' => $instance->domain,
+                    'e' => \get_class($e),
+                    'm' => $e->getMessage(),
+                ]);
```

Comment:
no need to throw up an error?

Reply:
nono if I throw that up then nothing has changed. The error can be catched here as this doesn't hinder the overall update process

Reply:
And actually catch the error higher in the chain..

Reply:
I think this is the right place. The instance should be created, but the name and the version are just not essential...